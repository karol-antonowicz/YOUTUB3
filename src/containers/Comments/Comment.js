import React from 'react';
import {Image, Button} from 'semantic-ui-react';
import {Rating} from '../../components/Rating/Rating';
import './Comment.scss';

export const Comment = () => {
    return (
        <div className='comment'>
            <Image className='user-image' src='https://via.placeholder.com/48x48' circular/>
            <div>
                <div className='user-name'>User Name</div>
                <span>Comment text</span>
                <div className='comment-actions'>
                    <Rating likeCount={1} dislikeCount={18} />
                    <Button size='mini' compact>REPLY</Button>
                </div>
            </div>
        </div>
    )
}