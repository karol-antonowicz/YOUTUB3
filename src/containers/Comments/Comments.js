import React from 'react';
import {CommentsHeader} from './CommentsHeader';
import {Comment} from './Comment';
import {AddComment} from './AddComment';
import './Comments.scss';

export const Comments = props => {
    return (
        <div className='comments'>
            <CommentsHeader amountComments={props.amountComments} />
            <AddComment />
            <Comment/>
            <Comment/>
            <Comment/>
            <Comment/>
            <Comment/>
        </div>
    );
}