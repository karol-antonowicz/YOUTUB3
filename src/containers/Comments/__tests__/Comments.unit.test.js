import React from 'react';
import {shallow} from 'enzyme';
import {Comments} from '../Comments';

describe('Comments', ()=>{
    test('Comments renders with no props', () => {
        const wrapper = shallow(<Comments/>);
        expect(wrapper).toMatchSnapshot();
    });
    test('Comments renders with props amountComments', () => {
        const wrapper = shallow(<Comments amountComments={245454}/>);
            expect(wrapper).toMatchSnapshot();
    });
})