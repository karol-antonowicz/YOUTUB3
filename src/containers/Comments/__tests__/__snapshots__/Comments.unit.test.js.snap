// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Comments renders with no props 1`] = `
<div
  className="comments"
>
  <CommentsHeader />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;

exports[`Comments Comments renders with props amountComments 1`] = `
<div
  className="comments"
>
  <CommentsHeader
    amountComments={245454}
  />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;
